backend/api/actions/datawork_import.py

```diff
@@ -17,22 +17,29 @@
 
 
 @transaction.atomic
-def datawork_import(*, importer):
+def datawork_import(*, dataset_checked, importer):
```

Comment:
If I understand correctly this is named like so because in the frontend this corresponds to datasets that have been checked on the list by the user ? Sending this name to the view is fine but here you should really use a meaningful name to the current context: `datasets_to_import` or `wanted_datasets`, something of the kind :wink: